staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission_test.go

```diff
@@ -46,6 +50,11 @@ func (f *fakeHookSource) ExternalAdmissionHooks() (*registrationv1alpha1.Externa
 	if f.err != nil {
 		return nil, f.err
 	}
+	for i, h := range f.hooks {
```

Comment:
Maybe tolerate the absence of this field instead?

Reply:
Do you mean the next line, "h.NamespaceSelector == nil"? It's mimicking the default value of `NamespaceSelector`.